test/user.js

```diff
@@ -0,0 +1,103 @@
+//During the test the env variable is set to test
```

Comment:
In particular here we are testing the user controller, so I like more to simulate the app folder structure and move this file inside the controllers folder. For example this file should be `test/controllers/user.js`